src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -174,31 +174,31 @@ public partial class HttpMethod : System.IEquatable<System.Net.Http.HttpMethod>
         public static System.Net.Http.HttpMethod Post { get { throw null; } }
         public static System.Net.Http.HttpMethod Put { get { throw null; } }
         public static System.Net.Http.HttpMethod Trace { get { throw null; } }
-        public bool Equals(System.Net.Http.HttpMethod other) { throw null; }
-        public override bool Equals(object obj) { throw null; }
+        public bool Equals(System.Net.Http.HttpMethod? other) { throw null; }
+        public override bool Equals(object? obj) { throw null; }
         public override int GetHashCode() { throw null; }
-        public static bool operator ==(System.Net.Http.HttpMethod left, System.Net.Http.HttpMethod right) { throw null; }
-        public static bool operator !=(System.Net.Http.HttpMethod left, System.Net.Http.HttpMethod right) { throw null; }
+        public static bool operator ==(System.Net.Http.HttpMethod? left, System.Net.Http.HttpMethod? right) { throw null; }
+        public static bool operator !=(System.Net.Http.HttpMethod? left, System.Net.Http.HttpMethod? right) { throw null; }
         public override string ToString() { throw null; }
     }
     public partial class HttpRequestException : System.Exception
     {
         public HttpRequestException() { }
-        public HttpRequestException(string message) { }
-        public HttpRequestException(string message, System.Exception inner) { }
-        public HttpRequestException(string message, System.Exception inner, System.Net.HttpStatusCode? statusCode) { }
+        public HttpRequestException(string? message) { }
+        public HttpRequestException(string? message, System.Exception? inner) { }
+        public HttpRequestException(string? message, System.Exception? inner, System.Net.HttpStatusCode? statusCode) { }
         public System.Net.HttpStatusCode? StatusCode { get { throw null; } }
     }
     public partial class HttpRequestMessage : System.IDisposable
     {
         public HttpRequestMessage() { }
-        public HttpRequestMessage(System.Net.Http.HttpMethod method, string requestUri) { }
-        public HttpRequestMessage(System.Net.Http.HttpMethod method, System.Uri requestUri) { }
-        public System.Net.Http.HttpContent Content { get { throw null; } set { } }
+        public HttpRequestMessage(System.Net.Http.HttpMethod method, string? requestUri) { }
+        public HttpRequestMessage(System.Net.Http.HttpMethod method, System.Uri? requestUri) { }
+        public System.Net.Http.HttpContent? Content { get { throw null; } set { } }
         public System.Net.Http.Headers.HttpRequestHeaders Headers { get { throw null; } }
         public System.Net.Http.HttpMethod Method { get { throw null; } set { } }
         public System.Collections.Generic.IDictionary<string, object> Properties { get { throw null; } }
```

Comment:
Should this be `IDictionary<string, object?>`?

Reply:
it could